.gitmodules

```diff
@@ -34,7 +34,7 @@
 	url = https://github.com/PX4/DriverFramework.git
 [submodule "src/lib/ecl"]
 	path = src/lib/ecl
-	url = https://github.com/PX4/ecl.git
+	url = https://github.com/lis-epfl/ecl.git
```

Comment:
We can't merge this like that, the ecl PR needs to be merged first and then the reference of the ecl submodule needs to be changed